import Exceptions
import fib

print(Exceptions.greet("Students"))
print(fib.fib(5))